mysite/settings.py

```diff
@@ -96,7 +96,7 @@
     DATABASES = {
         'default': {
             'ENGINE': 'django.db.backends.mysql',
-            'HOST': '/cloudsql/<your-cloud-sql-instance>',
+            'HOST': '/cloudsql/<your-project-id>:<your-cloud-sql-instance>',
```

Comment:
Hm... could we do a format string here that reference constants that are defined up top? (I assume these are to be filled in by the user? I figure it's more obvious when all the user-editable fields are up top)